web/cypress/e2e/logging/logs-adminConsole-ObserveLogs-admin.cy.ts

```diff
@@ -0,0 +1,283 @@
+    cy.getByTestId(TestIds.LogsQueryInput)
+      .find('textarea')
+      .clear()
+      .type('{{}log_type="infrastructure"{}}|json|log_source="container"', { delay: 0 })
+      .then(() => {
+        cy.getByTestId(TestIds.ExecuteQueryButton)
+         .click()
+         .then(() => {
+            cy.assertLogInLogsTable();
+         });
+      })
+  });
+
+  it('admin can display infra node logs',{tags:['@smoke','observ']}, () => {
+    cy.getByTestId(TestIds.TenantToggle).click();
+    cy.get('#logging-view-tenant-dropdown')
+      .contains('button', 'infrastructure')
+      .click();
+    cy.getByTestId(TestIds.ShowQueryToggle).click();
+    cy.getByTestId(TestIds.LogsQueryInput)
+      .find('textarea')
+      .clear()
+      .type('{{}log_type="infrastructure"{}}|json|log_source="node"', { delay: 0 })
+      .then(() => {
+        cy.getByTestId(TestIds.ExecuteQueryButton)
+          .click()
+          .then(() => {
+            cy.assertLogInLogsTable();
+        });
+      });
+  });
+
+  it('admin can display audit logs',{tags:['@smoke','observ']}, () => {
+    cy.getByTestId(TestIds.TenantToggle).click();
+    cy.get('#logging-view-tenant-dropdown')
+      .contains('button', 'audit')
+      .click();
+    cy.getByTestId(TestIds.ShowQueryToggle).click();
+    cy.getByTestId(TestIds.LogsQueryInput)
+      .find('textarea')
+      .clear()
+      .type('{{}log_type="audit"{}}', { delay: 0 })
+      .then(() => {
+        cy.getByTestId(TestIds.ExecuteQueryButton)
+          .click()
+          .then(() => {
+            cy.assertLogInLogsTable();
+          });
+      });
+  });
+
+  it('query with the selected namespaces',{tags:['@smoke','observ']}, () => {
+    const namespaces=[APP_NAMESPACE1, APP_NAMESPACE2]
+
+    cy.getByTestId(TestIds.AttributeFilters).within(() => {
+      cy.getByTestId(TestIds.AvailableAttributes).click();
+      cy.contains('Namespaces').click({ force: true });
+    })
+    cy.get('[data-test="AttributeOptions"] .pf-v5-c-menu-toggle__button').click();
+    cy.get('.pf-v5-c-menu.lv-plugin__search-select').within(() => {
+      namespaces.forEach(ns => {
+        cy.contains('label.pf-v5-c-menu__item', ns)
+          .find('input[type="checkbox"]')
+          .check({ force: true });
+        });
+    });
+    cy.get('[data-test="AttributeOptions"] .pf-v5-c-menu-toggle__button').click(); //close menu
+    cy.getByTestId(TestIds.ShowQueryToggle).click({force: true});
+    cy.getByTestId(TestIds.LogsQueryInput)
+      .find('textarea')
+      .invoke('val')
+      .then((val) => {
+        expect(val).to.include(APP_NAMESPACE1);
+        expect(val).to.include(APP_NAMESPACE2);
+      });
+
+    const pattern = new RegExp(`${APP_NAMESPACE1}|${APP_NAMESPACE2}`);
+    cy.getByTestId(TestIds.LogsTable).within(() => {
+      //  Click the first expand button
+      cy.get('#expand.pf-v5-c-table__td.lv-plugin__table__expand.pf-v5-c-table__toggle', { timeout: 6000 })
```

Comment:
same here, it seems we are mixing pf6 and pf5 selectors. A better way would be to add a test id to the logs table so we can reference it directly.